docs/src/examples/DecayTrain.tsx

```diff
@@ -9,19 +9,20 @@ import {
   Gesture,
   GestureDetector,
   GestureHandlerRootView,
+  PanGesture,
```

Comment:
To importing types we should use this syntax:
```suggestion
import type { PanGesture } 'react-native-gesture-handler';
```
Can you apply it also to the rest of occurs?

Reply:
yes sir